backend/models/price_list.py

```
@@ -55,7 +55,7 @@ class InsuranceProvider(Base):
 
     # Status
     is_active = Column(Boolean, default=True)
-    created_at = Column(DateTime, default=datetime.utcnow)
+    created_at = Column(DateTime, default=lambda: datetime.now(timezone.utc))
```

Ruff (0.15.11)

[error] Undefined name `timezone`

(F821)